tools/crm_rule.8.inc

```diff
@@ -1,7 +1,7 @@
 [synopsis]
-crm_rule mode [options]
+crm_rule <mode> [options]
 
-/state of rules/
+/from the Pacemaker configuration/
 .SH OPTIONS
 
 /multiple times/
```

Comment:
Looking at the generated man page after applying this patch, I still see:
```
NOTICE
   Application Options:
       -$, --version
              Display software version and exit

       -V, --verbose
              Increase debug output (may be specified multiple times)

NOTICE
AUTHOR
       Written by Andrew Beekhof and the Pacemaker project contributors
```

I don't know what all the NOTICE is about.

Reply:
yeah I didn't go too deep into the issues, just took care of some trivial things

Reply:
Oh... just get rid of the `/multiple times/` block.  That must be for something that no longer exists, since there's nothing in the help output that looks like a notice.

Reply:
thanks, done